### PR TITLE
uploads the python packages

### PR DESCRIPTION
**Pull request by uchinda-sph**

### CODE REVIEW

**Review by rh-sph — APPROVED** [review]

File: lambda.tf [hunk 1]
```diff
@@ -10,7 +10,9 @@ module "lambda" {
   memory_size = 128
   timeout     = 30
 
-  source_path = "${path.module}/src/lambda_function"
+  create_package         = false
+  local_existing_package = "${path.module}/src/lambda_function/lambda_function.zip"
```
Comment (smoneyan) [inline lambda.tf:14]: Are we building the file locally and creating this zip file ?

Comment (uchinda-sph) [reply lambda.tf:14]: @smoneyan Yes cause im getting following error on deployment 
`RuntimeError: Python interpreter version equal to defined lambda runtime (python3.7) should be available in system PATH`

Comment (uchinda-sph) [reply lambda.tf:14]: @smoneyan or else i can set the interpreter by external data resource 
```
data "external" "archive_prepare" {
  program = [local.python, "${path.module}/package.py", "prepare"]
  interpreter = "/usr/bin/python3.7" 
} 
```

**Review by smoneyan — APPROVED** [review]

File: lambda.tf (hunk 1 above)
Comment (smoneyan) [reply lambda.tf:14]: for these cases, what i've done in the past is .. use a dummy zip file for lambda creation and use GHA for CICD. using TF for building becomes a bit difficult sometimes :)

**Pull request merged**